public/sass/components/_submenu.scss

```diff
@@ -8,15 +8,6 @@
   padding: 0 0 $space-sm 0;
 }
 
-.annotation-segment {
```

Comment:
❓ I couldn't find the relationship between this class and the changes on the PR, is this just a deprecating class you are removing?

Reply:
yeah similar to above, i searched the entire codebase for `annotation-segment` and nowhere is using the class name.